test/e2e/lib/gutenberg/blocks/dynamic-separator-block-component.js

```diff
@@ -12,6 +12,47 @@ class DynamicSeparatorBlockComponent extends GutenbergBlockComponent {
 	static blockTitle = 'Dynamic HR';
 	static blockName = 'coblocks/dynamic-separator';
 	static blockFrontendSelector = By.css( '.entry-content .wp-block-coblocks-dynamic-separator' );
+
+	/**
+	 * Expands the horizontal ruler by the amount of pixels specified. This is useful
+	 * to test the functionality of the Dynamic HR in the editor and later in the
+	 * frontend too.
+	 *
+	 * @param {number} pixels the amount of pixerls to expand the HR ruler by, horizontally.
+	 */
+	async resizeBy( pixels ) {
+		// We need to move focus away from the layout grid, or any subsequent blocks inserted will be part of it
+		const resizeHandleSelector = await this.driver.findElement(
+			By.css(
+				`div[id='${ this.blockID.slice(
```

Comment:
`div[id='${ this.blockID.slice( 1 ) }']` can be just `this.blockID` (see [this comment](https://github.com/Automattic/wp-calypso/pull/45796#discussion_r500856393)).